flash/core/utilities/imports.py

```diff
@@ -99,5 +98,31 @@ def _compare_version(package: str, op, version) -> bool:
     _MATPLOTLIB_AVAILABLE,
     _COCO_AVAILABLE,
     _FIFTYONE_AVAILABLE,
+    _PYSTICHE_AVAILABLE,
 ])
 _SERVE_AVAILABLE = _FASTAPI_AVAILABLE and _PYDANTIC_AVAILABLE and _CYTOOLZ_AVAILABLE and _UVICORN_AVAILABLE
+
+_EXTRAS_AVAILABLE = {
+    'image': _IMAGE_AVAILABLE,
+    'tabular': _TABULAR_AVAILABLE,
+    'text': _TEXT_AVAILABLE,
+    'video': _VIDEO_AVAILABLE,
+    'serve': _SERVE_AVAILABLE,
+}
```

Comment:
enum?

Reply:
Wouldn't do it here since a mapping is perfectly fine when we map strings to boolean